spec/system/submissions/review_spec.rb

```diff
@@ -884,7 +884,66 @@
       sign_in_user school_admin.user,
                    referrer: review_timeline_event_path(submission_pending)
 
-      expect(page).to have_text("The page you were looking for doesn't exist!")
+      # Admin can access the page, but cannot review the submission.
+
+      expect(page).to have_text("Submission #1")
+
+      within("div[aria-label='submissions-overlay-header']") do
+        expect(page).to have_content("Submitted by #{student.name}")
+        expect(page).to have_link(
+          student.name,
+          href: "/students/#{student.id}/report"
+        )
+        expect(page).to have_link(target.title, href: "/targets/#{target.id}")
+        expect(page).to have_content(target.title)
+        expect(page).to have_text "Assigned Coaches"
+
+        # Hovering over the avatar should reveal the name of the assigned coach.
+        page.find("svg", text: "JD").hover
+        expect(page).to have_text("John Doe")
+      end
+
+      click_button "Start Review"
+
+      expect(page).to have_text("Authorization failed")
```

Comment:
Same complaints as in `spec/system/cohorts/student_report_spec.rb`:

1. Don't test for _errors_. This option shouldn't be shown to a school admin!
2. Scenario looks unnecessarily large.